test/unit/nrm/behaviours/save-application.test.js

```diff
@@ -51,7 +49,7 @@ describe('/apps/nrm/behaviours/save-application', () => {
       Base.prototype.getValues.restore();
     });
 
-    it('when the data is saved there is a value in sessionModel[\'application-id\']', async() => {
+    xit('when the data is saved there is a value in sessionModel[\'application-id\']', async() => {
```

Comment:
`xit`?

Reply:
`xit` is like `it`, except it skips/disables the test.

Reply:
I know but is this intentional?